.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release Artifacts
```

Comment:
this isn't relevant to docs. It's only to publish to PyPI.

Reply:
right - maybe this is morphing a little more into a "setup docs AND release automation" PR. Is that okay?

Reply:
I personally like to keep my PRs small and focused. It allows for better code review. I'd encourage the release automation being a dedicated PR since they are 100% orthogonal: you can add one totally separately from docs.

Reply:
okay makes sense - I'll remove it from this PR.